drinfomon.lic

```diff
@@ -873,7 +873,7 @@ exp_hook = proc do |server_string|
   when regex_exp_columns
     if parsing_exp_output
       server_string.scan(regex_exp_columns) do |skill_value, rank_value, percent_value, rate_value|
-        rate = learning_rates.index(rate) # convert word to number
+        rate_value = learning_rates.index(rate_value) # convert word to number
```

Comment:
I'm not a fan of reassigning to "input" parameters (loop param in this case). Maybe `rate_as_word` on line 875 and `rate_as_number` on line 876?